pkg/emitter/nats_emitter_test.go

```diff
@@ -0,0 +1,256 @@
+//
+// Copyright 2022 The GUAC Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package emitter
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/guacsec/guac/internal/testing/dochelper"
+	nats_test "github.com/guacsec/guac/internal/testing/nats"
+	"github.com/guacsec/guac/pkg/handler/processor"
+	"github.com/guacsec/guac/pkg/logging"
+	"github.com/nats-io/nats.go"
+	uuid "github.com/satori/go.uuid"
+)
+
+var (
+	// Taken from: https://slsa.dev/provenance/v0.1#example
```

Comment:
Probably fine for this test, but the examples under slsa.dev aren't full examples of the SLSA document. For example the digests aren't real digests `{"sha256": "5678..."}`

Reply:
Yup at this level of test it does not matter. For the larger integration test a valid doc that can be ingested will be used.